man/systemd.exec.xml

```diff
@@ -1129,6 +1129,18 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         security.</para></listitem>
```

Comment:
not that it matters much, but the commit title should probably include `=`, i..e. say something like this:

```
core: ProtectHostname= feature 
```

(background: across our codebase we try to expose settings to users with a `=` suffix to always clarify that this is a setting of some kind)